src/simulations/multirun.jl

```diff
@@ -0,0 +1,87 @@
+export multirun!
+Vector_or_Tuple = Union{AbstractArray, Tuple}
+
+"""
+    multirun!(models::Vector, agent_step!, model_step!, n; kwargs...)
+Call [`run!`](@ref) for all `model ∈ models`. Each `model` should be a (different) instance
+of an [`AgentBasedModel`](@ref) but probably initialized with a different random seed or
+different initial agent distribution. All models obey the same rules `agent_step!, model_step!`.
+
+Similarly to [`run!`](@ref) this function will collect data. It will furthermore
+add one additional column to the dataframe called `:replicate`, which will have
+integer values counting the replicate simulations.
+The function returns `agent_df, model_df, models`.
+
+The keyword `parallel=false` will run the simulations in parallel using Julia's
```

Comment:
= true

Reply:
It should be `false` by default, because noone would define everything prefaced with `@everywhere` by default.

Reply:
Agreed, but then you need to change the wording of the text. Right now it says that false runs the simulations in parallel.

Reply:
okay I'm stupid.